### ISSUE TITLE
How to finetune?

### ISSUE THREAD

**Issue by Erick-hub** [liweileev/FET-GAN#4]
Hi, thank you for your paper code!

I would like to test the pretrained networks on custom text effects and fonts. So far I haven't been getting the desired results with new styles. How could I finetune the pretrained models ?

**Comment by liweileev** [comment]
To be honest, the finetune for new text effects is tricky, and it does have some flaws.

First of all, it is required that the new effect is similar to at least one of the pre-trained text effect to some extent. Only in this way, G and E can learn the features of the new text effect faster; otherwise, the initialization of finetuning networks will hardly help.

Secondly, the phenomenon of overfitting is very easy to occur. Therefore, when doing experiments, I will reduce the learning rate as much as possible, and count finetune by iterations instead of using epoch (although as a temporary dataset established by few-shot manner, the sample size of an epoch is not much). In extreme cases, we even output the result every iteration to see if overfitting occurs and adjust the learning rate accordingly.

In addition, from the experimental experience, it is much easier to finetune the new font than the text effect. I guess it is AdaIN's first-order moment feature representation, which simplifies the feature too much, especially for complex effects and styles. So compared to simpler fonts, the style dimensions of text effect objects are more diverse, and the finetune effect is also worse.
